### ISSUE TITLE
Syslog UDP input does not set gl2_remote_ip in Graylog 3.0

### ISSUE THREAD

**Issue by jplitza** [Graylog2/graylog2-server#5264]
## Expected Behavior

The `gl2_remote_ip` field gets populated with the source IP address for messages sent in via SyslogUDPInput.

## Current Behavior

It doesn't. A pipeline containing
```
debug($message.gl2_remote_ip);
```
logs
```
INFO  [Function] PIPELINE DEBUG: Passed value is NULL.
```

## Steps to Reproduce (for bugs)

1. Run Graylog 3.0.0-2.alpha.3 from the Ubuntu repository
2. Set up Syslog UDP input
3. Send in some messages

## Context

Same problem as #3980, but for Syslog instead of GELF.

## Your Environment

* Graylog Version: 3.0.0-alpha.3+7530b69, codename Space Moose
* Elasticsearch Version: 5.6.12
* MongoDB Version: 3.6.3-0ubuntu1
* Operating System: Ubuntu 18.04
* Browser version: Firefox 63.0

**Comment by jplitza** [comment]
Actually, this doesn't only affect Syslog, but also GELF, and only occurs in Graylog 3.0.0. Downgradaing to 2.4 "solved" the problem.

**Comment by bernd** [comment]
@jplitza Thanks for the report. This is the same issue as in #5293 and will be fixed before the final 3.0 release.

**Comment by bernd** [comment]
@jplitza This will work again once #5293 is fixed. I am closing this issue, please check #5293 for updates. Thank you!
